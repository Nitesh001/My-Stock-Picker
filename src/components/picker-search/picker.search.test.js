import React from 'react';
import renderer from 'react-test-renderer';
import PickerSearch from './picker.search';

test('Search shows placeholder, default input and two buttons', () => {
  const component = renderer.create(
    <PickerSearch placeholder="placeholder" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test('Search shows custom value entered by user', () => {
  const component = renderer.create(
    <PickerSearch placeholder="placeholder" value="value" />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});