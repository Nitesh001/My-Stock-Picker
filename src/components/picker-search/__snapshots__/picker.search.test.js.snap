// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search shows custom value entered by user 1`] = `
<div
  className="mb-3 input-group"
>
  <input
    className="form-control"
    onChange={[Function]}
    onFocus={[Function]}
    onKeyUp={[Function]}
    placeholder="placeholder"
    type="text"
    value="value"
  />
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Search
  </button>
  <button
    className="btn btn-secondary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Reset
  </button>
</div>
`;

exports[`Search shows placeholder, default input and two buttons 1`] = `
<div
  className="mb-3 input-group"
>
  <input
    className="form-control"
    onChange={[Function]}
    onFocus={[Function]}
    onKeyUp={[Function]}
    placeholder="placeholder"
    type="text"
    value=""
  />
  <button
    className="btn btn-primary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Search
  </button>
  <button
    className="btn btn-secondary"
    disabled={false}
    onClick={[Function]}
    type="button"
  >
    Reset
  </button>
</div>
`;
